xedisk.d

```diff
@@ -457,7 +457,7 @@ void add(string[] args)
 	enforce(args.length >= 4, format(
 		"Missing arguments. Try `%s help add'.", args[0]));
 
-	scope stream = new FileStream(File(args[2], "r+b"));
```

Comment:
May not be sufficient. DMD seems to get destruction order wrong when any `scope` class instance is initialized with anything else than plain `new` expression. See WA in ccd2cdf.